sdk/lib/opentelemetry/sdk/trace/export/multi_span_exporter.rb

```diff
@@ -29,7 +29,7 @@ def export(spans)
               begin
                 merge_result_code(result_code, span_exporter.export(spans))
               rescue => e # rubocop:disable Style/RescueStandardError
-                logger.warn("exception raised by export - #{e}")
+                OpenTelemetry.logger.warn("exception raised by export - #{e}")
```

Comment:
Do you know why `logger` doesn’t work here? It seems awkward to “namespace” as `OpenTelemetry.logger`. Is this a `require` order issue, or do I misunderstand the way the method lookup works?

Reply:
OK, so yeah, I misunderstood the lookup here. Constant lookup follows module nesting (except when it doesn't). Method lookup only follows the inheritance chain, and module nesting is irrelevant.